src/JuMP.jl

```diff
@@ -395,6 +396,36 @@ function add_bridge(model::Model,
     return
 end
 
+function _moi_add_bridge(model::Nothing,
+                        BridgeType::Type{<:MOI.Bridges.AbstractBridge})
+    # No optimizer is attached, the bridge graph will be printed if one is attached
+    return
+end
+function _moi_print_bridge_graph(io::IO, model::MOI.ModelLike)
+    error("Cannot print bridge graph if `bridge_constraints` was set to `false` in the",
+          " `Model` constructor.")
+end
+function _moi_print_bridge_graph(io::IO, bridge_opt::MOI.Bridges.LazyBridgeOptimizer)
+    MOI.Bridges.print_graph(io, bridge_opt)
+    return
+end
+function _moi_print_bridge_graph(io::IO, caching_opt::MOIU.CachingOptimizer)
+    _moi_print_bridge_graph(io, caching_opt.optimizer)
+    return
+end
+
+"""
+     print_bridge_graph(model::Model)
+
+Print out the bridge graph explaining which bridges and in what order they are used to bridge constrained function or variable
```

Comment:
Wrap to 80 characters and end with a full stop.

Maybe just:
```
Print a summary of the bridges added to the model, explaining which constraints 
they bridge and in what order.
```

Reply:
I think I made a mistake the correct statement should be
`Print graph created by added bridges in `model::Model.``

Reply:
I think the correct statement should be
`Print a summary of the bridges added to the model, explaining which constraint they bridge.`

Reply:
It should be the plural `constraints`.